src/common/constants.js

```diff
@@ -2800,252 +2883,243 @@ const days = {
   9: thirtyDays,
   10: thirtyOneDays,
   11: thirtyDays,
-  12: thirtyOneDays
+  12: thirtyOneDays,
 };
 
-const dependentRelationships = [
-  'Daughter',
-  'Son',
-  'Stepson',
-  'Stepdaughter',
-  'Father',
-  'Mother',
-  'Spouse',
-  'Other'
-];
+const dependentRelationships = ['Daughter', 'Son', 'Stepson', 'Stepdaughter', 'Father', 'Mother', 'Spouse', 'Other'];
```

Comment:
Not sure why this array is being moved into a single line. I don't think it should be modified.

Is there a max character limit per line set (I forget what the lint setting is named)?

Reply:
Yes it looks like that was changed to one line because it did not hit the char limit of 120 chars. I can make that shorter, or look for that rule and disable it, or leave it be. I don't really have an opinion to be honest I think this is just Prettier Standard.

Reply:
Hmm, I thought that was set to 80 everywhere else.